src/si/volume_rate.rs

```diff
@@ -97,6 +97,8 @@ quantity! {
             "zeptoliters per second";
         @yoctoliter_per_second: prefix!(milli) * prefix!(yocto); "yL/s", "yoctoliter per second",
             "yoctoliters per second";
+        @liter_per_minute: prefix!(milli) / 6.0_E1; "L/min", "liter per minute",
+            "liters per minute";
```

Comment:
One minor formatting fix and I'll merge!

```suggestion

        @liter_per_minute: prefix!(milli) / 6.0_E1; "L/min", "liter per minute",
            "liters per minute";
```

Reply:
Done!